addons/sale_project/static/src/components/project_right_side_panel/components/project_profitability.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="sale_project.ProjectProfitability" t-inherit="project.ProjectProfitability" t-inherit-mode="extension">
+        <xpath expr="//a[hasclass('revenue_section')]" position="before">
+            <t t-if="revenue.isFolded and revenue.isFolded !== undefined">
```

Comment:
It's there to prevent the button from appearing on section that are not linked to any sol (like other_cost or purchase). It is possible to have items in the revenues section but that item does not have the isFolded param set. I've just put it on both button by default, but its true that it's useless on the first if.